pom.xml

```diff
@@ -5,7 +5,7 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>1.609.1</version>
+    <version>2.25</version>
```

Comment:
This changes the minimum compatibility version of Jenkins if I am not mistaken, is this required?